Registry/RegistryParse.py

```diff
@@ -384,22 +405,38 @@ def validate_checksum(self):
         """
         return self.calculate_checksum() == self.checksum()
 
+    def validate(self):
+        """
+        Are the file checksum and sequence numbers valid?
+        Obsolete, use recovery_required instead.
+        """
+        from warnings import warn
+        warn("validate is obsolete, use recovery_required instead!")
+        return self.validate_checksum() and self.validate_sequence_numbers()
+
     def recovery_required(self):
         """
         Are the file checksum and sequence numbers valid?
-        Return a tuple with two boolean values:
-          - the first one is True when the REGF block recovery is required,
-          - the second one is True when data recovery is required.
+        Return a named tuple with two boolean values:
+          - the recover_header is True when the REGF block recovery is required,
+          - the recover_data is True when data recovery is required.
         """
+        recover = namedtuple('recover', ['recover_header', 'recover_data'])
+
         if not self.validate_checksum():
             # Header is invalid, this also implies data recovery
-            return (True, True)
+            recover.recover_header = True
```

Comment:
since namedtuples are immutable, this shouldn't normally work. recommend something like:

    return RecoveryStatus(True, True)